tests/python/relax/test_transform_rewrite_cuda_graph.py

```diff
@@ -0,0 +1,211 @@
+                    ax0 = T.axis.spatial(T.int64(8), i0_fused_0 * T.int64(8) + i0_fused_1)
+                    T.reads(rxplaceholder[ax0], rxplaceholder_1[()])
+                    T.writes(T_add[ax0])
+                    T_add[ax0] = rxplaceholder[ax0] + rxplaceholder_1[()]
+
+    @T.prim_func
+    def pad(rxplaceholder: T.Buffer[T.int64(8), "float32"], PadInput: T.Buffer[T.int64(10), "float32"]):
+        # function attr dict
+        T.func_attr({"tir.noalias": True, "global_symbol": "pad"})
+        # body
+        # with T.block("root")
+        for i0_fused_0 in T.thread_binding(T.int64(1), thread="blockIdx.x"):
+            for i0_fused_1 in T.thread_binding(T.int64(10), thread="threadIdx.x"):
+                with T.block("PadInput"):
+                    i0 = T.axis.spatial(T.int64(10), i0_fused_0 * T.int64(10) + i0_fused_1)
+                    T.reads(rxplaceholder[i0 - T.int64(1)])
+                    T.writes(PadInput[i0])
+                    PadInput[i0] = T.if_then_else(T.int64(1) <= i0 and i0 < T.int64(9), rxplaceholder[i0 - T.int64(1)], T.float32(1), dtype="float32")
+
+    @R.function
+    def cuda_graph_capture_func_alloc() -> R.Tuple(R.Tensor((2, 4), dtype="float32"), R.Tensor((8,), dtype="float32"), R.Tensor((8,), dtype="float32")):
+        # block 0
+        gv: R.Tensor((2, 4), dtype="float32") = R.builtin.alloc_tensor((2, 4), dtype="float32", runtime_device_index=0)
+        gv1: R.Tensor((8,), dtype="float32") = R.builtin.alloc_tensor((8,), dtype="float32", runtime_device_index=0)
+        gv2: R.Tensor((8,), dtype="float32") = R.builtin.alloc_tensor((8,), dtype="float32", runtime_device_index=0)
+        gv3: R.Tuple(R.Tensor((2, 4), dtype="float32"), R.Tensor((8,), dtype="float32"), R.Tensor((8,), dtype="float32")) = (gv, gv1, gv2)
+        return gv3
+
+    @R.function
+    def cuda_graph_capture_func_capture(allocs: R.Tuple(R.Tensor((2, 4), dtype="float32"), R.Tensor((8,), dtype="float32"), R.Tensor((8,), dtype="float32"))) -> R.Tuple():
+        # block 0
+        alloc: R.Tensor((2, 4), dtype="float32") = allocs[0]
+        alloc1: R.Tensor((8,), dtype="float32") = allocs[1]
+        alloc2: R.Tensor((8,), dtype="float32") = allocs[2]
+        _1: R.Tuple() = reshape(alloc, alloc1)
+        gv0: R.Tensor((), dtype="float32") = R.const(1, "float32")
+        _2: R.Tuple() = add(alloc1, gv0, alloc2)
+        gv4: R.Tuple() = R.tuple()
+        return gv4
+
+    @R.function
+    def main(x: R.Tensor((2, 4), dtype="float32")) -> R.Tensor((10,), dtype="float32"):
+        # block 0
+        gv: R.Tuple(R.Object, R.Tuple(R.Tensor((2, 4), dtype="float32"), R.Tensor((8,), dtype="float32"), R.Tensor((8,), dtype="float32"))) = R.call_builtin_with_ctx("vm.builtin.get_captured_cuda_graph", (cuda_graph_capture_func_alloc, cuda_graph_capture_func_capture), sinfo_args=R.Tuple(R.Object, R.Tuple(R.Tensor((2, 4), dtype="float32"), R.Tensor((8,), dtype="float32"), R.Tensor((8,), dtype="float32"))))
```

Comment:
need a test case of what it CANNOT handle, for example, a function that also takes opaque object and mix such operation in between, we should make sure we get multiple sub-regions, and opaque execution stays as it is